app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -28,6 +19,55 @@ def translator(action, _config)
           ]
         end
       end
+
+      # Docker orb specific implementation
+      class DockerOrb
+        def initialize
+          @config = {}
```

Comment:
this is not really necessary as the class doesn't need to keep any internal state